src/main.rs

```diff
@@ -73,18 +74,11 @@ fn main() {
                 ),
         )
         .arg(
-            Arg::with_name("wasm-remote-insecure")
-                .env("CHIMERA_WASM_REMOTE_INSECURE")
```

Comment:
If we don't care a lot about setting this behaviours through envvars, I think we can move to `clap_app!` macro, that would reduce a lot of repetition here. I couldn't find how to configure the name of environment variables bound to args though, so we would lose reading args from environment variables.

Reply:
Yes, as discussed let's remove it. I think it's better to handle that inside of a dedicated configuration file